app/main/plugins/core/basic-apps/linux.js

```diff
@@ -0,0 +1,71 @@
+import { remote } from 'electron'
+import path from 'path'
+import fs from 'fs'
+import uniq from 'lodash/uniq'
+import { shellCommand } from 'cerebro-tools'
+
+let appDirs = [
+  path.join(remote.app.getPath('home'), '.local', 'share'),
+  path.join('usr', 'share'),
+  path.join('usr', 'share', 'ubuntu'),
+  path.join('usr', 'share', 'gnome'),
+  path.join('usr', 'local', 'share'),
+  path.join('var', 'lib', 'snapd', 'desktop')
```

Comment:
unfortunately I don't, but I managed to do it the hard way, through dev tools :)

```
appDirs: 
["/home/maximbaz/.local/share/applications", "usr/share/applications", "usr/share/ubuntu/applications", "usr/share/gnome/applications", "usr/local/share/applications", "var/lib/snapd/desktop/applications", "/tmp/.mount_IKkzVw/usr/share/applications", "share/applications", "applications"]

DIRECTORIES:
["/home/maximbaz/.local/share/applications"]
```

I apply this command in debugger just before DIRECTORIES are set:
`appDirs = appDirs.map(dir => dir[0] == '/' ? dir : ('/' + dir));`

Then appDirs and DIRECTORIES are fixed:

```
appDirs = ["/home/maximbaz/.local/share/applications", "/usr/share/applications", "/usr/share/ubuntu/applications", "/usr/share/gnome/applications", "/usr/local/share/applications", "/var/lib/snapd/desktop/applications", "/tmp/.mount_IKkzVw/usr/share/applications", "/share/applications", "/applications"]

DIRECTORIES = ["/home/maximbaz/.local/share/applications", "/usr/share/applications"]
```